Consul.Test/AgentTest.cs

```diff
@@ -1025,6 +1025,34 @@ public async Task Agent_Metrics()
             Assert.NotNull(agentMetrics.Response.Samples);
         }
 
+        [Fact]
+        public async Task Agent_CARoots()
+        {
+            var caRoots = await _client.Agent.GetCARoots();
+            Assert.NotEqual((ulong)0, caRoots.LastIndex);
+            Assert.NotNull(caRoots.Response.ActiveRootID);
+            Assert.Equal("11111111-2222-3333-4444-555555555555.consul", caRoots.Response.TrustDomain);
+            Assert.Single(caRoots.Response.Roots);
+            var root = caRoots.Response.Roots.First();
+            Assert.NotNull(root.ID);
+            Assert.NotNull(root.Name);
+            Assert.NotNull(root.SigningKeyID);
+            Assert.NotNull(root.ExternalTrustDomain);
+            Assert.NotNull(root.NotBefore);
+            Assert.NotNull(root.NotAfter);
+            Assert.NotNull(root.RootCert);
+            Assert.Null(root.IntermediateCerts);
+            Assert.True(root.Active);
+            Assert.NotNull(root.PrivateKeyType);
+            Assert.NotEqual(0, root.PrivateKeyBits);
```

Comment:
I think this one needs to go inside the `if (AgentVersion >= SemanticVersion.Parse("1.7.0"))` block.

Reply:
Yeah, a bit surprising, but should be working now, fixed it!